src/Service/JiraApiService.php

```diff
@@ -130,6 +133,38 @@ public function getCurrentUserPermissions(): mixed
         return $this->get(self::API_PATH_MY_PERMISSIONS);
     }
 
+    /**
+     * Get all projects, including archived.
+     *
+     * @return SprintReportProjects array of SprintReportProjects
+     *
+     * @throws ApiServiceException
+     */
+    public function getAllProjectsV2(): SprintReportProjects
+    {
+        return new SprintReportProjects();
```

Comment:
These should be implemented.

Reply:
"These"? createProject and what else?

Reply:
getAllProjectsV2
getProjectV2
getProjectVersions